README.md

```diff
@@ -18,17 +18,34 @@ Last meeting
   - AdaConv P672 
 
 - Convolution Layers - Lin & Qiming
-  For concolution layer, we implement batch normalization layer to obey the convolutional property. It is a method of unifying scattered data and a method of optimizing neural networks, so that different elements of the same feature map, at different locations, are normalized in the same way. With unified specifications of data, machine learning can be more easily learned into the data Rules. We also apply spatial softmax layer is used to find the expected pixel location of feature maps.
+  # introducing cnn structure
+  # explain the reason of applying bn and spacial softmax
+  In a neural network, each hidden unit’s input distribution changes every time there is a parameter update in the previous layer. This is called the internal covariate shift. This makes training slow and requires a very small learning rate and a good parameter initialization. This problem is solved by normalizing the layer’s inputs over a mini-batch and this process is therefore called Batch Normalization. We also implement spatial softmax from TensorFlow API, a function returns the expected pixel locations of each feature map. It can be used as weights to compute the mean pixel locations of each feature.
```

Comment:
- We are introducing our own CNN model here so we don't need to bring in BN and Softmax as a solution to solve general CNN problems. You may need to rewrite this paragraph and describe our model in details. 
- We don't usually mention any API while introducing a CNN model. We just say something like: we are using xxx for each convolution layer because ... 
- Missing description on our model
- Missing description on spatial softmax